qemu/tests/virtio_serial_hotplug_max_chardevs.py

```diff
@@ -94,8 +95,7 @@ def run_bg_test():
     try:
         serial_devices[0].hotplug(vm.monitor, vm.devices.qemu_version)
     except QMPCmdError as e:
-        if "Duplicate ID '%s' for device" % serial_devices[0] not in str(
-                e.data):
+        if not (re.search("Duplicate (device| )ID '%s'" % serial_devices[0], str(e.data))):
```

Comment:
Could you test with it ?   Logically, it should be `(device |)ID`, not `(device| )ID`.